src/main/java/WayofTime/alchemicalWizardry/common/rituals/RitualEffectUnbinding.java

```diff
@@ -101,7 +101,20 @@ else if (UnbindingRegistry.isRequiredItemValid(itemStack))
                     item.setDead();
                     doLightning(world, x, y, z);
                     ItemStack[] inv = ((BoundArmour) itemStack.getItem()).getInternalInventory(itemStack);
-
+                    int bloodSockets = 0;
```

Comment:
Instead of chaining if/else-ifs, why not handle this via a switch/case?

If not that, what about storing the amount of sockets needed in a map keyed by the ModItems.BOUND_ITEM?

Basically, you'd then only have to do the following:

```
Map<Item, int> socketAmounts = buildSocketAmounts();

// ...

bloodSockets = socketAmounts.get(itemStack.getItem());
```

You'd basically just want to insert the number of sockets needed into the map, keyed by the item.  If there's no match, just set bloodSockets to 0.